pkg/galera/galera.go

```diff
@@ -60,25 +61,27 @@ func (g *GaleraState) Compare(other GaleraRecoverer) int {
 	return 0
 }
 
-func (g *GaleraState) Marshal() ([]byte, error) {
+func (g *GaleraState) MarshalText() ([]byte, error) {
```

Comment:
Seems like a good idea to implement `TextMarshaler` and `TextUnmarshaler`, more idiomatic